benchmarks/gravity_thin_shell/gravity_thin_shell.prm

```diff
@@ -0,0 +1,82 @@
+# A simple setup for testing the gravity postprocessor using the 
+# sample scheme 'map'.
+
+# General parameters
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output_thin_shell_gravity
+set Nonlinear solver scheme                = no Advection, no Stokes
+
+# Model geometry
+subsection Geometry model
+  set Model name = spherical shell
+  subsection Spherical shell
+    # layer 0km depth
+    #set Inner radius = 6366e3
+    #set Outer radius = 6376e3
+    # layer 100km depth
+    set Inner radius = 6266e3
+    set Outer radius = 6276e3
+    # layer 500km depth
+    #set Inner radius = 5866e3
+    #set Outer radius = 5876e3
+    # layer 1500km depth
+    #set Inner radius = 4866e3
+    #set Outer radius = 4876e3
+    # layer 3000km depth
+    #set Inner radius = 3366e3
+    #set Outer radius = 3376e3
+    set Custom mesh subdivision = number of slices
+    set Number of slices = 1
+    set Initial lateral refinement = 5
+  end
+end
+
+# Model boundary velocity
```

Comment:
All of these section-beginning comment lines are identical to the section names themselves, so they can be removed.

Reply:
done